reverse_engineering/helpers/bigQueryHelper.js

```diff
@@ -7,13 +7,15 @@ const createBigQueryHelper = (client, log) => {
 
 	const getPrimaryKeyConstraintsData = async (projectId, datasetId) => {
 		try {
-			return await client.query({
-				query: `SELECT * 
+			return (
+				await client.query({
+					query: `SELECT * 
 				FROM ${projectId}.${datasetId}.INFORMATION_SCHEMA.KEY_COLUMN_USAGE AS KCU
 				INNER JOIN ${projectId}.${datasetId}.INFORMATION_SCHEMA.TABLE_CONSTRAINTS AS TC
 				USING(constraint_name)
 				WHERE TC.constraint_type = "PRIMARY KEY";`,
-			});
+				})
+			)[0];
```

Comment:
Is it safe to get the first element of an array? Is there a case this query can return an empty array? If yes, it can lead to unexpected error...